docs/src/app/experiments/rtl.module.css

```diff
@@ -0,0 +1,239 @@
+.rtl {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.row {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 2rem;
+}
+
+.IconButton {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 2.5rem;
+  height: 2.5rem;
+  padding: 0;
+  margin: 0;
+  outline: 0;
+  border: 1px solid var(--color-gray-200);
+  border-radius: 0.375rem;
+  background-color: var(--color-gray-50);
+  color: var(--color-gray-900);
+  user-select: none;
+
+  @media (hover: hover) {
+    &:hover {
+      background-color: var(--color-gray-100);
+    }
+  }
+
+  &:active {
+    background-color: var(--color-gray-100);
+  }
+
+  &[data-popup-open] {
+    background-color: var(--color-gray-100);
+  }
+
+  &:focus-visible {
+    outline: 2px solid var(--color-blue);
+    outline-offset: -1px;
+  }
+}
+
+.Icon {
+  width: 1.25rem;
+  height: 1.25rem;
+}
+
+.Positioner {
+  &:focus-visible {
+    outline: 0;
+  }
+}
+
+.Popup {
+  box-sizing: border-box;
+  padding: 1rem 1.5rem;
+  border-radius: 0.5rem;
+  background-color: canvas;
+  color: black;
+  transform-origin: var(--transform-origin);
+  transition-property: transform, opacity;
+
+  &[data-open]:not([data-starting-style]),
+  &[data-ending-style] {
+    transition-duration: 150ms;
+  }
+
+  &[data-side='inline-end'] {
+    transform: scale(1) translateX(-16px);
+  }
+
+  &[data-side='inline-start'] {
+    transform: scale(1) translateX(16px);
+  }
+
+  &[data-starting-style],
+  &[data-ending-style] {
+    opacity: 0;
+
+    &[data-side='inline-end'] {
+      transform: scale(0.9) translateX(16px);
+    }
+
+    &[data-side='inline-start'] {
+      transform: scale(0.9) translateX(-16px);
+    }
+  }
```

Comment:
@atomiks Having either physical only or logical sides in the rendered data attr works for these, but it does make more sense for it to be consistent with the prop that was provided 
CC @vladmoroz

Reply:
Yeah, it has to be consistent with the provided prop